app/assets/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -362,6 +390,7 @@ export function parseNml(nmlString: string): Promise<TreeMapType> {
               timestamp: Date.now(),
               edges: new EdgeCollection(),
               isVisible: _parseFloat(attr, "color.a") !== 0,
+              groupId: groupId >= 0 ? groupId : DEFAULT_GROUP_ID,
```

Comment:
I had to do it this way, because DEFAULT_GROUP_ID is null and not a number. Flow obviously complains, because the return value of _parseInt should be a number. If you change that return value to ?number you run into all kinds of other type problems :/
The groupId -1 is currently used in the tree_hierarchy_view as a placeholder for trees without a groupId, so using that in an NML file would probably break stuff, yes.